src/d/d_wood.cpp

```diff
@@ -4,184 +4,970 @@
 //
 
 #include "d/d_wood.h"
+#include "JAZelAudio/JAIZelBasic.h"
+#include "JSystem/J3DGraphBase/J3DShape.h"
+#include "JSystem/J3DU/J3DUClipper.h"
+#include "JSystem/JMath/JMath.h"
+#include "JSystem/JUtility/JUTAssert.h"
+#include "SSystem/SComponent/c_bg_s_gnd_chk.h"
+#include "SSystem/SComponent/c_lib.h"
+#include "d/d_bg_s_gnd_chk.h"
+#include "d/d_bg_s_spl_grp_chk.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_drawlist.h"
+#include "d/d_kankyo_wether.h"
+#include "d/d_procname.h"
+#include "d/d_tree.h"
+#include "dolphin/gf/GFGeometry.h"
+#include "dolphin/gf/GFTev.h"
+#include "dolphin/gf/GFTransform.h"
+#include "dolphin/gx/GXAttr.h"
+#include "dolphin/gx/GXDisplayList.h"
 #include "dolphin/types.h"
+#include "m_Do/m_Do_lib.h"
+#include "m_Do/m_Do_mtx.h"
+
+// These belong in the d_tree compilation unit. But in order to match, they must
+// be in this compilation unit so they can have a 16-bit offset, allowing us
+// load their pointer using a single load immediate instruction
+// u8 g_dTree_shadowTexCoord[2];
+
+//-----------------------------------------
+// Types
+//-----------------------------------------
+typedef u8 ALIGN_DECL(32) jData;
```

Comment:
what does the name "jData" mean? is this official or did you name it?

Reply:
Ah, that was something I was experimenting with as a way to avoid manualy setting the alignment of several of the large types such as display lists. I named it. 

What do you think? Is it worth keeping around and renaming to something better?